sourcegraph/cody
# Disable gzip compression for SSE streams

Closes #2223

@rafax: Are your server-side fixes for this already rolled out somewhere so I can test this?

## Test plan

- Use a request interceptor (i.e Proxyman)
- Notice the new headers being added when doing e.g. a code completion

<img width="1212" alt="Screenshot 2023-12-11 at 10 56 56" src="https://github.com/sourcegraph/cody/assets/458591/de2bfff4-3654-4178-ab0b-47a35b018adb">
